sdk/communication/arm-communication/README.md

```diff
@@ -6,7 +6,6 @@ REST API for Azure Communication Services
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/communication/arm-communication) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/arm-communication) |
-[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-communication?view=azure-node-preview) |
```

Comment:
The comment here was to remove this part ?view=azure-node-preview , not the entire line. please fix